core/mono-samples/iOS/Program.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
should this be `net6.0`? it probably should, but it's possible the dotnet sdk doesn't support it yet.

we might want to mention in the readme that we require a daily master build of the dotnet sdk from https://github.com/dotnet/installer.

Reply:
I tried `net6.0` a while ago, and I believe the dotnet sdk did not support it at that time.
Will add the note about the dotnet sdk build.

Reply:
Updated it to be `net6.0`.